### PR TITLE
Refactor/OL74-auth_reducers

### PR DESCRIPTION
**Pull request by CavalcanteLucas**
**Link to task:** 
https://labcodes.atlassian.net/browse/OL-74

**How to test:**
- Check if failure message is appearing at login form when providing invalid credentials;
- Check if there is no confirmation e-mail sent on user registration;

**Description of your solution:**
This PR refactors the variable names from `mapStateToProps` and `mapDispatchToProps` over the `auth` front-end to keep the pattern introduced with the set of components for `todo_list`. The actions and reducers files have also been reorganized to ease the reading. This PR also fixes errors in the front and backend of the `auth` app.

### CODE REVIEW

File: react-app/src/js/auth/actions.js [hunk 1]
```diff
@@ -116,34 +129,41 @@ export const clearPasswordResetSuccessMessage = () => ({
...
 
-export const clearRegisterSuccessMessage = () => ({
-  type: CLEAR_REGISTER_SUCCESS_MESSAGE
+export const clearPasswordResetConfirmSuccessMessage = () => ({
+  type: CLEAR_PASSWORD_RESET_CONFIRM_SUCCESS_MESSAGE
 });
 
-export const clearRegisterFailureMessage = () => ({
-  type: CLEAR_REGISTER_FAILURE_MESSAGE
+export const clearPasswordResetConfirmErros = () => ({
```
Comment (lucianoratamero) [inline react-app/src/js/auth/actions.js:162]: typo here!

File: react-app/src/js/auth/components/PasswordResetConfirmForm.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { connect } from "react-redux";
 
 import {
   passwordResetConfirm,
-  clearPasswordResetConfirmFailureMessage,
+  clearPasswordResetConfirmErros,
```
Comment (lucianoratamero) [inline react-app/src/js/auth/components/PasswordResetConfirmForm.js:8]: and here!

File: react-app/src/js/auth/components/PasswordResetConfirmForm.js [hunk 3]
```diff
@@ -20,18 +20,18 @@ export class PasswordResetConfirmForm extends React.Component {
   static propTypes = {
     passwordResetConfirm: PropTypes.func.isRequired,
     history: PropTypes.object,
-    successMessage: PropTypes.string,
-    errors: PropTypes.object,
+    passwordResetConfirmSuccessMessage: PropTypes.string,
+    passwordResetConfirmErrors: PropTypes.object,
     match: PropTypes.object.isRequired,
-    clearPasswordResetConfirmFailureMessage: PropTypes.func.isRequired,
+    clearPasswordResetConfirmErros: PropTypes.func.isRequired,
```
Comment (lucianoratamero) [inline react-app/src/js/auth/components/PasswordResetConfirmForm.js:26]: here

File: react-app/src/js/auth/components/PasswordResetConfirmForm.js [hunk 4]
```diff
@@ -40,17 +40,21 @@ export class PasswordResetConfirmForm extends React.Component {
...
+      setSuccessMessage(passwordResetConfirmSuccessMessage);
       history.push("/");
     }
   }
 
   componentWillUnmount() {
-    const { clearPasswordResetConfirmFailureMessage } = this.props;
-    clearPasswordResetConfirmFailureMessage();
+    const { clearPasswordResetConfirmErros } = this.props;
+    clearPasswordResetConfirmErros();
```
Comment (lucianoratamero) [inline react-app/src/js/auth/components/PasswordResetConfirmForm.js:57]: here

File: react-app/src/js/auth/components/PasswordResetConfirmForm.js [hunk 5]
```diff
@@ -124,15 +130,16 @@ export class PasswordResetConfirmForm extends React.Component {
...
+  passwordResetConfirmErrors: state.auth.passwordResetConfirmErrors
 });
 
 const mapDispatchToProps = dispatch => ({
   passwordResetConfirm: (uid, token, password1, password2) =>
     dispatch(passwordResetConfirm(uid, token, password1, password2)),
-  clearPasswordResetConfirmFailureMessage: () =>
-    dispatch(clearPasswordResetConfirmFailureMessage()),
+  clearPasswordResetConfirmErros: () =>
+    dispatch(clearPasswordResetConfirmErros()),
```
Comment (lucianoratamero) [inline react-app/src/js/auth/components/PasswordResetConfirmForm.js:142]: here

File: react-app/src/js/auth/actions.js (hunk 1 above)
Comment (CavalcanteLucas) [reply react-app/src/js/auth/actions.js:162]: i don't understand, how so ?

Comment (CavalcanteLucas) [reply react-app/src/js/auth/actions.js:162]: i was blind. now i can see.
https://www.youtube.com/watch?v=wWUC2-ypnDQ

Comment (CavalcanteLucas) [reply react-app/src/js/auth/actions.js]: fixed

File: react-app/src/js/auth/components/PasswordResetConfirmForm.js (hunk 2 above)
Comment (CavalcanteLucas) [reply react-app/src/js/auth/components/PasswordResetConfirmForm.js]: ty, fixed!

File: react-app/src/js/auth/components/PasswordResetConfirmForm.js (hunk 4 above)
Comment (CavalcanteLucas) [reply react-app/src/js/auth/components/PasswordResetConfirmForm.js]: ty, fixed!

File: react-app/src/js/auth/components/PasswordResetConfirmForm.js (hunk 3 above)
Comment (CavalcanteLucas) [reply react-app/src/js/auth/components/PasswordResetConfirmForm.js]: ty, fixed!

File: react-app/src/js/auth/components/PasswordResetConfirmForm.js (hunk 5 above)
Comment (CavalcanteLucas) [reply react-app/src/js/auth/components/PasswordResetConfirmForm.js]: ty, fixed!

**Pull request merged**